src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -57,7 +57,7 @@ const SidebarSidecarLayoutContent = ({
   }
 
   const SidecarContent = (): ReactElement => {
-    if (sidecarSlot) {
+    if (typeof sidecarSlot !== 'undefined') {
```

Comment:
I think we want to pass `null` to `sidecarSlot` to keep it empty, so updated this check to be a bit more precise.